### ISSUE TITLE
Request headers tab went missing with 1.9.6

### ISSUE THREAD

**Issue by jimmybergman** [graphql/graphiql#2484]
In 1.9.5 it looked like:
<img width="265" alt="Screenshot 2022-06-07 at 10 28 45" src="https://user-images.githubusercontent.com/429413/172334505-6b17ccc9-9abe-4c51-9945-add7fffa67d9.png">

In 1.9.6 only the query variables is available instantiated as
```
      ReactDOM.render(
        React.createElement(GraphiQL, {
          fetcher,
          defaultVariableEditorOpen: true,
          tabs: true,
        }),
        document.getElementById('graphiql'),
      )
```

**Comment by thomasheyenbrock** [comment]
Introduced in https://github.com/graphql/graphiql/pull/2461/commits/81ad808ce27e88f7b380d3c33da9685f8d18941f

**Comment by thomasheyenbrock** [comment]
Fixed in 1.9.8
